source/docs/yearly-overview/yearly-changelog.rst

```diff
@@ -1,203 +1,202 @@
-- Removed deprecated and broken SPI methods
-- Removed 2-argument constructor to ``SlewRateLimiter``
-- Removed ``frc2::PIDController`` alias (``frc::PIDController`` already existed)
-- For ease of use, ``loadAprilTagFieldLayout()`` now throws an unchecked exception instead of a checked exception
-- Add new parameter for ``ElevatorSim`` constructor for starting height
-- Report error on negative PID gains
-
-## Simulation
-
-- Unified PWM simulation Speed, Position, and Raw values to be consistent with robot behavior
-- Expanded DutyCycleEncoderSim API
-- Added ability to set starting state of mechanism sims
-- Added mechanism-specific SetState overloads to physics sims
+- The units library has been refactored to have unit-specific measurement classes instead of a single generic ``Measure`` class. The new measurement classes have clearer names (``Distance`` instead of ``Measure<Distance>``, or ``LinearAcceleration`` instead of ``Measure<Velocity<Velocity<Distance>>>``), and implement math operations to return the most specific result types possible instead of a wildcard ``Measure<?>``.
+- Add persistent alerts API. Alerts are displayed on supported dashboards such as Shuffleboard and Elastic.
+- Add LED pattern API for easily animating addressable LEDs
+- Breaking: Remove deprecated ``Gyro`` and ``Accelerometer`` interface
+- Breaking: Remove deprecated ``Notifier.SetHandler`` function
+- Remove ``RobotInit`` usage in examples. Use constructor instead. RobotInit may be deprecated in the future.
+- Deprecate ``AxisCamera``
+- C++: Add ``FRC_ReportWarning``
+- Implement ``Sendable`` for HID classes
+- Include sendable type information in topic metadata
+- ``GenericHID.setRumble``: Fix Java integer overflow
+
+#### Commands
+
+- Breaking: Remove deprecated ``CommandBase``
+- Remove deprecated ``TrapzoidProfileCommand`` API
+- Breaking: Remove deprecated C++ method ``TransferOwnership``
+- Deprecate ``PIDCommand``, ``PIDSubsystem``, ``ProfiledPIDCommand``, ``ProfiledPIDSubsystem``, ``TrapezoidProfileSubsystem``
+- Deprecate ``TrapezoidProfileCommand``. Use :doc:`TrapezoidProfile Directly </docs/software/commandbased/profile-subsystems-commands>`
+- Deprecate proxy supplier constructor
+- Cache controller ``BooleanEvents`` / ``Triggers`` and directly construct ``Triggers``, fixing issues if ``BooleanEvents`` / ``Triggers`` are created in loops
+- Add deadband trigger methods to ``CommandGenericHID``
+- Make requirements private
+- Add ``setRumble`` and ``isConnected`` to ``CommandGenericHID``
+- Add ``StartRun`` command factory
+- Rename ``deadlineWith`` to ``deadlineFor``
+- Fix double composition error message truncation
+
+#### NetworkTables
+
+- Server round robin message processing
+- Unify NetworkInterface and MessageHandler
+- Add StopRead/StartRead to WireConnection
+- Client: only connect to IPv4 addresses
+- ParallelTcpConnector: Add option to resolve only IPv4 addresses
+- Deprecate setNetworkTablesFlushEnabled
```

Comment:
Several of these are internal only and should be removed from this list (Unify NetworkInterface and MessageHandler, Add StopRead/StartRead, ParallelTcpConnector).

Reply:
Thanks. I gave up trying to figure out which ones should be listed or not